server/hotrod/src/test/scala/org/infinispan/server/hotrod/HotRodSingleClusteredNonLoopbackTest.scala

```diff
@@ -38,7 +39,11 @@ class HotRodSingleClusteredNonLoopbackTest extends MultipleCacheManagersTest {
    @Test(enabled=false) // Disable explicitly to avoid TestNG thinking this is a test!!
    override def createBeforeClass() {
       super.createBeforeClass()
-      val iface = findNetworkInterfaces(false).next
+      val ifaces = findNetworkInterfaces(false)
+      var iface: NetworkInterface = null
+      do {
+         iface = ifaces.next
+      } while (!iface.getInetAddresses.hasMoreElements)
```

Comment:
I think you can do `val iface = findNetworkInterfaces(false).filter(_.getInetAddresses.hasMoreElements).next()`